bin/ga_script.sh

```diff
@@ -129,8 +129,8 @@ python $GA_OG_ANALYTICS \
     $GA_CLIENT_SECRET 68455797 $GA_PORTAL_INI \
     $STARTYR-$CURMTH-01 $PRVYR-$PRVMTH-$LASTDY download \
     >> $GA_LOG_FILE 2>&1
-mv $GA_TMP_DIR/od_ga_downloads.xls $GA_ARCHIVE_DIR/downloads-$PRVMTH$STARTYR-$PRVMTH$PRVYR.xls
-UPLOADFILE=$GA_ARCHIVE_DIR/downloads-$PRVMTH$STARTYR-$PRVMTH$PRVYR.xls
+mv $GA_TMP_DIR/od_ga_downloads.xls $GA_ARCHIVE_DIR/openDataPortal.siteAnalytics.downloads-$PRVMTH$STARTYR-$PRVMTH$PRVYR.xls
```

Comment:
@wardi I think if I add that prefix pattern to the 2 xls files your suggestion would work

Reply:
this won't work unless we rename all the old files too, right?